pennylane/pauli/pauli_word_sparse_matrix.md

```diff
@@ -0,0 +1,119 @@
+# Pauli word sparse representation
+
+Pauli words, a.k.a. Pauli strings, are operatos resulting from the tensor products of Pauli matrices or the identity. Namely:
+$$
+I = \begin{pmatrix}1 & 0\\ 0 & 1\end{pmatrix}, \ \  X = \begin{pmatrix}0 & 1\\ 1 & 0\end{pmatrix}, \ \  Y = \begin{pmatrix}0 & -i\\ i & 0\end{pmatrix}, \ \ Z = \begin{pmatrix}1 & 0\\ 0 & -1\end{pmatrix}.
+$$
```

Comment:
Random fly-by comment, is the latex supposed to render in the "view file" view?

Reply:
I am not sure 🤔 I expected it to do so, since it does render latex in notebook markdown cells, but it is not working for me. The first solution I found was to use ````math ``` instead of $ $, but this does not render in markdown.

Reply:
I fixed it! GitHub does not deal well with commands before or after $$ or \\, so I had to introduce spaces and newlines.